ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.html

```diff
@@ -616,7 +616,14 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
                             </ng-container>
                           </ng-container>
                           <ng-container
-                            *ngIf="col.type === 'DATE'">{{getUtil().getDateFormatted(onc[col.participantColumn.name])}}
+                            *ngIf="col.type === 'DATE'">
+                            {{
+                            onc[col.participantColumn.name]?.split('-').length > 2 ?
+                              (onc[col.participantColumn.name]  | date: 'MM/dd/yyyy') :
+                            onc[col.participantColumn.name]?.split('-').length === 2 ?
+                              (onc[col.participantColumn.name] | date: 'MM/yyyy') :
+                              (onc[col.participantColumn.name] | date: 'yyyy')
+                            }}
```

Comment:
It's quite complex conditional logic and I'd suggest moving it into the component method or a computed property, rather keeping it in the template

Reply:
Yeah, I am aware of it. It is just temporary. Will be improved later.